docs/samples/logger/basic/sklearn-logging.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: "serving.kubeflow.org/v1alpha2"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris"
+spec:
+  default:
+    predictor:
+      minReplicas: 1      
+      logger:
+        url: http://message-dumper.default/
+        mode: all
+      sklearn:
+        storageUri: "gs://kfserving-samples/models/sklearn/iris"
+        resources:
```

Comment:
Why not just let this default? 0.1 is quite restrictive

Reply:
ok. Just as these are demos can easily use up cluster capacity. But probably cleaner.